### ISSUE TITLE
[Downloads Manager] Raycast cannot find the Downloads folder when using two macbooks with Raycast Cloud Sync

### ISSUE THREAD

**Issue by rajath-ramakrishna-cruise** [raycast/extensions#15480]
### Extension

https://www.raycast.com/thomas/downloads-manager

### Raycast Version

1.86.0

### macOS Version

Sonoma 14.7

### Description

Using Downloads Manager on two different machines with two different usernames and that share the same Raycast config (using Cloud Sync) leads to one of them not being able to locate the Downloads folder.

So, setting the Downloads folder on one machine would make the other machine be unable to find the path to it and vice versa.

### Steps To Reproduce

1. Get two macbooks with different usernames
2. Use Raycast sync on both
3. Install Downloads Manager extension
4. Set Downloads folder on Macbook 1
5. Go to Macbook 2 and try to use the Downloads folder and you'll be asked to set it. If you set it, then the next time use try to use Downloads folder on Macbook 1, you'll be asked to set it again.

### Current Behaviour

![image](https://github.com/user-attachments/assets/6cc18775-f77e-400e-a5eb-dad870a786c7)

This is the window you get when you try to access the Downloads folder on one machine when its location was set on another machine.

### Expected Behaviour

Downloads folder must be accurately located on both machines, possibly by just following the `~/Downloads` and not expanding the `~` because expanding that would include the username in the path (which would be different on two different machines) and I believe that's what is creating the conflict.

**Comment by pernielsentikaer** [comment]
This should soon be resolved if it's not already is, could you have a look @rajath-ramakrishna-cruise

**Comment by rajath-ramakrishna-cruise** [comment]
Yes, this has been fixed. Thanks much! Appreciate the quick resolution. 🙂
